Resources/Prototypes/_Goobstation/MartialArts/kravmaga.yml

```
@@ -0,0 +1,50 @@
+- type: entity
+  id: ActionLegSweep
+  name: Leg Sweep
+  description: Sweeps the legs out from under a target to knock them down for a few seconds.
+  categories: [ HideSpawnMenu ]
+  components:
+  - type: InstantAction
```

YAMLlint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)